src/run.ts

```diff
@@ -127,6 +127,8 @@ export async function run(argv: Args) {
     default:
       clack.log.error(`No setup wizard selected!`);
   }
+
+  await runPrettierIfInstalled();
```

Comment:
h: let's move this call inside the individual run*wizard functions. If we leave it here, the prompt would be rendered after the outro messages which looks a bit off.

Reply:
@Lms24, moved the prettier runs inside `nextjs`, `remix`, `svelte` and `react-native` wizards.